<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Add_loan</class>
 <widget class="QMainWindow" name="Add_loan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>LibSys</string>
  </property>
  <property name="windowIcon">
   <iconset theme="QIcon::ThemeIcon::UserAvailable"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>150</y>
      <width>361</width>
      <height>231</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLineEdit" name="search_client">
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
       <property name="placeholderText">
        <string>Wyszukaj</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTableView" name="client_table">
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>180</y>
      <width>271</width>
      <height>151</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Verdana</family>
     </font>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QComboBox" name="loan_book_box">
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
       <property name="placeholderText">
        <string>Wybierz książkę</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
         <property name="text">
          <string>Data wypożyczenia</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="loan_loan_date">
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
         <property name="text">
          <string>Data zwrotu</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateEdit" name="loan_return_date">
         <property name="font">
          <font>
           <family>Verdana</family>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QPushButton" name="loan_add_button">
       <property name="font">
        <font>
         <family>Verdana</family>
        </font>
       </property>
       <property name="text">
        <string>Wypożycz</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
